e2e/test/scenarios/visualizations/line_chart.cy.spec.js

```diff
@@ -415,8 +415,13 @@ describe("scenarios > visualizations > line chart", () => {
       cy.findByText("Category is Doohickey");
     });
 
-    it.skip("should not drop the chart legend (metabase#4995)", () => {
+    it("should not drop the chart legend (metabase#4995)", () => {
       cy.findAllByTestId("legend-item").should("contain", "Doohickey");
+
+      cy.log("Ensure that legend is hidden when not dealing with multi series");
+      cy.findByTestId("viz-settings-button").click();
+      cy.findByTestId("remove-CATEGORY").click();
+      cy.get("main main").should("not.contain", "Doohickey");
```

Comment:
> `cy.get("main main")`

Should we define a function in `e2e-ui-elements-helpers.js` to give this a more semantic name (e.g. `visualization()`)?

Reply:
It's not a bad idea. I was actually surprised one didn't exist already lol. I could move it and name it something like `queryBuilderMain()`